flink-kubernetes-operator-autoscaler/src/main/java/org/apache/flink/kubernetes/operator/autoscaler/config/AutoScalerOptions.java

```diff
@@ -87,28 +88,28 @@ private static ConfigOptions.OptionBuilder autoScalerConfig(String key) {
     public static final ConfigOption<Integer> VERTEX_MAX_PARALLELISM =
             autoScalerConfig("vertex.max-parallelism")
                     .intType()
-                    .defaultValue(Integer.MAX_VALUE)
+                    .defaultValue(200)
                     .withDescription(
                             "The maximum parallelism the autoscaler can use. Note that this limit will be ignored if it is higher than the max parallelism configured in the Flink config or directly on each operator.");
 
     public static final ConfigOption<Double> MAX_SCALE_DOWN_FACTOR =
             autoScalerConfig("scale-down.max-factor")
                     .doubleType()
-                    .defaultValue(0.6)
+                    .defaultValue(1.0)
```

Comment:
Curious why we choose to loose it to 1.0. We found in that TPR tends to be overestimated a lot and leading to overly aggressive downscaling when:

- the pipeline is underloaded and far from optimal.
- Avg CPU allocated per slot is < 1.

The reason is that linear scaling with busy time metrics assumes no resource competition between tasks when we pushing up the loads, however, when avg CPU allocated per slot is small, resource competition beween tasks will be more and more severve as we pushing up the overall loads of the pipeline.

Reply:
I tend to agree here, I think 0.6 is not a bad default. I could also see 0.7 to be good as well but I would not go higher than that because often a large scale down leads to an immediate scale up due to non-linearity

Reply:
I've seen situations where vertices wouldn't be able to scale down all the way to 1 if they were idle. Maybe setting it to `0.5` would still allow that. From a theoretical perspective this makes sense because a halving (0.5) would ensure that you get from parallelism 2 to 1.

Reply:
What I don't understand is why `0.6` is worse than 0.5 :)

Reply:
The code looks like it works as expected. I've started a test pipeline to verify is this is still an issue.

@X-czh In my use case there is a CPU per slot, so I might be less affected by this. In any case, we probably keep 0.6 as the default here.